src/main/resources/assets/carpet-addons-not-found/lang/en_us.json

```diff
@@ -18,6 +18,7 @@
   "carpet.rule.netheritePickaxeInstantMineEndStone.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine end stone type blocks.",
   "carpet.rule.netheritePickaxeInstantMineNetherBricks.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine nether brick type blocks.",
   "carpet.rule.phantomsObeyHostileMobCap.desc": "Phantoms will no longer spawn if the hostile mobcap is full. This is per player.",
+  "carpet.rule.pickFlowersFromPots.desc": "Right-clicking on a flowerpot with a flower in it will put the flower into your invantory",
```

Comment:
Description should be "Right-clicking on a flowerpot containing a flower will always put the flower into your inventory."